### ISSUE TITLE
Can i write wallet   address   in the  source?

### ISSUE THREAD

**Issue by carpediem01** [fireice-uk/xmr-stak-cpu#351]

**Comment by psychocrypt** [comment]
First sry for the following answere because the will answere your question
but this will not help you.
Yes it is possible because everything which is created at runtime can also
predefined in source code.
This requires C++ hacking skills.

Am 26.09.2017 3:04 Nachm. schrieb "carpediem01" <notifications@github.com>:

**Comment by sunk818** [comment]
@carpediem01 what does this question mean? are you asking about changing donation address to something else? Or do you mean you want change the pool details in config.txt?

**Comment by scarlion1** [comment]
Yes i would like to know if the donation address can be defined/modified somewhere?  Reason being that i will run the miner for a non-profit and will be using their wallet address, so if i could change donation address to my wallet then it can help offset my costs.  Thanks

**Comment by nowheel** [comment]
if you want change address on source, for the fee, remember, you don't develop this software... the developer must eat. sorry for my eng but it's is not my primary lamguage.
I see for change address, but the developer doesn't use it for fee. use other mining pool (i think it's private, i don't see this. correct me if wrong), i write a simplescript for autocompile in ubuntu, but never i think to change it, for respect to developer. 
i repeat, the developer must eat don't change the ccde of fee. thank you to developer for make  and update xmr-stak(cpu/gpu)

**Comment by Strongwatchman** [comment]
I apologize for being a noob here...in advance. 
I am wondering if it's possible to work with the developer of the XMR-STAK Unified Miner?
I would like to white label the GUI miner and promote it widely. 
Of course I want the Developer to eat...and mad RESPECT...so keep the dev in there....but is it possible say....for the sake of marketing effort for someone like myself to get paid for promoting this miner? We would take promotion in a similar fashion to the developer....and I would need the developer to add another "dev mining address" to mine to our own pool. 

We would be promoting the XMR-STAK-Unified GUI miner on all the appropriate algorithms. For Cryptonight....and related coins. Is this possible? Can I talk to or contact the dev?

I am on Telegram and Facebook as Greg Eddolls
We have big plans for this if we can get it white labeled and have access to front end GUI. 
We can change the basics...but I want to honor the developer here. 

Thanks
